nodebuilder/store.go

```diff
@@ -150,6 +154,66 @@ type fsStore struct {
 	dirLock *flock.Flock // protects directory
 }
 
+// Finds a single running node and return its store path.
+// Node Type Order: Bridge, Full, Light
+// Network Order: Mainnet, Mocha, Arabica, Private
+func DiscoverOpened() (string, error) {
+	nodeTypes := []nodemod.Type{nodemod.Bridge, nodemod.Full, nodemod.Light}
+	defaultNetwork := []p2p.Network{p2p.Mainnet, p2p.Mocha, p2p.Arabica, p2p.Private}
+
+	for _, t := range nodeTypes {
+		for _, n := range defaultNetwork {
+			path, err := DefaultNodeStorePath(t.String(), n.String())
+			if err != nil {
+				return "", err
+			}
+
+			ok, _ := IsOpened(path)
+			if ok {
+				return path, nil
+			}
+		}
+	}
+
+	return "", nil
```

Comment:
I think we should error here and wrap it in getStorePath

Reply:
wrapped